src/components/Table/index.tsx

```diff
@@ -169,7 +169,6 @@ function Table<D, K extends string | number, C extends Column<D, K, any, any>>(
                             <Renderer
                                 {...otherProps}
                                 className={_cs(cellRendererClassName, styles.cellComponent)}
-                                name={id}
```

Comment:
This will most probably break type.
We should also remove `name` from `BaseCell` interface and Omit on props